src/main/java/gov/cabinetoffice/gapfindapiadmin/controllers/ApiKeyController.java

```diff
@@ -33,28 +34,34 @@ public class ApiKeyController {
     public static final String REVOKE_API_KEY_CONFIRMATION_PAGE = "revoke-api-key-confirmation";
     public static final String ERROR_PAGE = "error-page";
     public static final String SUPER_ADMIN_PAGE = "super-admin-api-keys";
-    public static final String SUPER_ADMIN_ROLE = "SUPER_ADMIN";
 
     private final ApiKeyService apiKeyService;
     private final ApiGatewayService apiGatewayService;
     private final PaginationHelper paginationHelper;
+    private final UserServiceConfig userServiceConfig;
 
     @GetMapping
     @PreAuthorize("hasAuthority('TECHNICAL_SUPPORT')")
     public ModelAndView showKeys() {
         final GrantAdmin grantAdmin = (GrantAdmin) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
         final String departmentName = grantAdmin.getFunder().getName();
 
-        return new ModelAndView(ORGANISATION_API_KEYS_PAGE)
+        ModelAndView model = new ModelAndView(ORGANISATION_API_KEYS_PAGE)
```

Comment:
finale 🤌

Reply:
i didn't add final, because depending on the isAdmin condition, we then may add more object to it